tests/tck/features/go/SampleLimit.feature

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2021 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+Feature: Sample and limit
+
+  Background: Prepare space
+    Given a graph with space named "nba"
+
+  Scenario: Sample Limit Go in One step
```

Comment:
```
GO FROM 'Tim Duncan' OVER like LIMIT 1; 

GO FROM 'Tim Duncan' OVER like LIMIT ["1"];
```

Please add tests as above, also the sample clause.

Reply:
Done